src/PaletteStore.ts

```diff
@@ -98,12 +98,11 @@ export class PaletteStore {
    * @return {JsonPaletteType} reference to the named palette, or undefined if
    *                           no such palette.
    */
-  async getNamedPalette(paletteName: string, loadFunction?: (file:string, path:string) => Promise<JsonPaletteType>): Promise<JsonPaletteType> {
+  async getNamedPalette(paletteName: string, loadFunction?: (file:string) => Promise<JsonPaletteType>): Promise<JsonPaletteType> {
```

Comment:
This modification requires a change to a test in `PaletteStore.test.ts`.  I am seeing a failure with a type error:

```
src/PaletteStore.test.ts:125:77 - error TS2345: 
Argument of type '(file: string, path: string) => Promise<JsonPaletteType>' is not assignable to 
parameter of type '(file: string) => Promise<JsonPaletteType>'.
      Target signature provides too few arguments. Expected 2 or more, but got 1.
```

Also, should the `file` argument be `filePath` since it is no longer just a file name, but the path to the file (without the extension)?